site/docs/docs/components/pagination.md

```diff
@@ -42,10 +42,10 @@ import { Alert } from "agnostic-vue";
 <summary class="disclose-title">View source</summary>
 
 ```jsx
-import { useEffect, useState } from 'react';
-import 'agnostic-react/dist/common.min.css';
-import 'agnostic-react/dist/esm/index.css';
-import { Pagination } from 'agnostic-react';
+import { useEffect, useState } from "react";
```

Comment:
Can you double check here on the double quotes? I'm pretty sure react is set to single quotes

Reply:
I'm not actually sure why this got changed at all, I must've accidentally messed with something I'll check it out, because I didn't mess with the react documentation at all

Reply:
oh I think eslint is applying this for some reason potentially